src/common/utils.cpp

```diff
@@ -632,6 +632,43 @@ void DecodeStringSignature(int8* signature, int8* target)
     strncpy((char*)target, (const char*)decodedSignature, sizeof decodedSignature);
 }
 
+void PackSoultrapperName(std::string name, uint8 output[], uint8 size)
```

Comment:
I'd like to see some comments here documenting intent.  Its clean, for style, but takes a lot of time to figure out whats going on.

Reply:
Added some very basic explanation of what the function does. It's in "it works well, but it looks funky, don't touch it!" mode